full-moon/src/ast/parser_util.rs

```diff
@@ -179,6 +179,12 @@ macro_rules! keep_going {
 #[doc(hidden)]
 #[macro_export]
 macro_rules! define_roblox_parser {
+    ($parser:ident, $node:ty, $mock_ty:ty, |_, $state:ident| $body:expr) => {
```

Comment:
Do we need this? `_` is a valid identity.

Reply:
I mainly copied this off how `define_parser!` macro works, it's the same layout there. AIUI, since this is a macro matching on a specific token, it's different

Reply:
Just tested this, and you're right - I'll remove this definition. Not sure if you want to do the same for `define_parser!`?

Reply:
Probably should.